galaxycache_test.go

```diff
@@ -369,48 +372,39 @@ func TestPeers(t *testing.T) {
 
 }
 
+// TODO(willg): Changed the order of assignment here to make this work, not
+// entirely sure of the purpose of this test but it basically still works as
+// it was
 func TestTruncatingByteSliceTarget(t *testing.T) {
 	var cacheFills AtomicInt
 	stringGalaxy, ctx, _ := testSetupStringGalaxy(&cacheFills)
-	var buf [100]byte
-	s := buf[:]
-	if err := stringGalaxy.Get(ctx, "short", TruncatingByteSliceSink(&s)); err != nil {
+	var buf ByteCodec
+	if err := stringGalaxy.Get(ctx, "short", &buf); err != nil {
 		t.Fatal(err)
 	}
+	s := buf[:]
```

Comment:
I'd use a cast here rather than a slice expression:
```suggestion
	s := []byte(buf)
```